doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/decision-tables-defining-proc.adoc

```diff
@@ -30,7 +30,19 @@ NOTE: As an alternative to populating both the object type and constraint cells,
 If you need to add more rule tables to this decision table spreadsheet, skip a row after the last rule in the previous table, label another `RuleTable` cell in the same column as the previous `RuleTable` and  `RuleSet` cells, and create the new table following the same steps in this section (rows 19-29 in example).
 . Save your XLS or XLSX spreadsheet to finish.
 
-NOTE: Only the first worksheet in a spreadsheet workbook will be processed as a decision table when you upload the spreadsheet in {CENTRAL}. Each `RuleSet` name combined with the `RuleTable` name must be unique across all decision table files in the same package.
+[NOTE]
+====
+
+Only the first worksheet in a spreadsheet workbook is processed as a decision table when you upload the spreadsheet in {CENTRAL}. Each `RuleSet` name combined with the `RuleTable` name must be unique across all decision table files in the same package.
```

Comment:
Minor Suggestion - rearrange the sentence so the user action is first:

s/Only the first worksheet in a spreadsheet workbook is processed as a decision table when you upload the spreadsheet in {CENTRAL}./When you upload a spreadsheet workbook in {CENTRAL}, only the first worksheet of the workbook is processed as a decision table.

Minor Suggestion - clarify the user interaction with {CENTRAL}:

s/in {CENTRAL}/to {CENTRAL}/using the {CENTRAL} UI

Reply:
@csherrar Thanks for reviewing!
how about changing the sentence as: _By default, only the first worksheet in a spreadsheet workbook is processed as a decision table when you upload the spreadsheet in the {CENTRAL}._ 
Since the default process of taking only first worksheet needs to be emphasized.